templates/zerver/help/create-a-stream.md

```diff
@@ -36,12 +36,12 @@ There are several parameters you can set while creating a stream. All but
 * **Stream description**: Helps users decide whether to subscribe when they
   are browsing streams.
 
-* **Who can access the stream?**: See [Stream permissions](/help/stream-permissions).
-
 * **Announce stream**: Posts a message to `#general` advertising the new
```

Comment:
I'd guess we need to update this "Announce stream" text to say "Announce new stream in"

Reply:
Yeah, good point!

@drrosa Do we have a good pattern for how to manage variables (in this context, some stream name that is specified by the org admins)? In addition to the challenge of what to put before the colon, I don't think the current text after the colon is ideal either.

Reply:
Hmm, this is the first time I encounter this particular pattern. Maybe we can say something like:

**Announce new stream in**: Posts an automated notice in the [creation announcements stream](/help/configure-notification-bot#new-stream-announcements) to advertise the new stream.

Reply:
Hm, we should keep the wording consistent with the settings option, which labels the announcement stream dropdown as "New stream announcements".

Maybe something like this?

**Announce new stream in `#<stream configured by your organization>`**: Posts an automated notice that the stream was created in the [new stream announcements stream](/help/configure-notification-bot#new-stream-announcements).

Sigh, how many times can we us the word "stream" in one sentence?

Reply:
Yeah, I think the text before the colon is good. I can't think of a different way to phrase the text after the colon and still keep the wording consistent without using the word "stream" multiple times. ^^